neural_network_lyapunov/control_affine_system.py

```diff
@@ -143,6 +144,179 @@ def compute_G_range_ia(self):
         return G_lo, G_up
 
 
+class SecondOrderControlAffineSystem(ControlPiecewiseAffineSystem):
+    """
+    A second-order system
+    q̇ = v
+    v̇ = a(x) + b(x)u
+    where the state is x = [q, v].
+    """
+    def __init__(self, x_lo, x_up, u_lo, u_up):
+        super(SecondOrderControlAffineSystem,
+              self).__init__(x_lo, x_up, u_lo, u_up)
+        assert (self.x_dim % 2 == 0)
```

Comment:
I guess there's no way to support orientations and keep the dynamics linear...

Reply:
No we can't support orientation.

I create this `ReLUSecondOrderAffineSystem` so that I can test my other code with a toy `ReLUSecondOrderAffineSystem` example. In practice later we probably will model inverted pendulum using this `ReLUSecondOrderAffineSystem` class. For quadrotor we will write new derived class.